### DISCUSSION TITLE
Punycode deprecation causing super-linter to fail

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#29029] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

We use [super-linter](https://github.com/super-linter/super-linter/) in our github actions to validate configurations of renovate. Recently it has started failing when using the renovate-config-validator binary in it's runs. 

Running locally allows us to validate that the config is valid, but it does produce a warning at the start of execution that shows renovate is using the deprecated punycode package. 

Is there an alternative as requested in the error logs attached that we can use? 

### Logs (if relevant)

<details><summary>Logs</summary>

```
(node:63652) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

```

</details>

**Comment by rarkins** [comment]
I don't see how this is a Renovate bug. punycode is used throughout a number of eslint packages, which Renovate uses in devDependencies.

**Reply by ghost** [reply]
Is this not a dependency within renovate-config-validator? (Limited understanding, so happy to be told I am wrong)

**Reply by rarkins** [reply]
That validator is built from within this same repo

**Comment by ghost** [comment]
I did not notice the migrateConfig notification within the linter, this fixed the issue. Thank you for your time.
